src/graceful_termination.jl

```diff
@@ -61,6 +64,34 @@ container hook. Typically, K8s initiates graceful termination via the `TERM` sig
 as Julia forcefully terminates when receiving this signal and Julia does not support
 user-defined signal handlers we utilize `preStop` instead.
 
+### Using superviser entrypoint
+
+K8sDeputy provides a bash script in bin/superviser.sh which handles the `TERM` signal and
+sends the `"terminate"` message to the graceful termination socket.  This can be used as the
+`ENTRYPOINT` for your docker image after installation as follows (assuming an Ubuntu/Debian
+base image):
+
+```dockerfile
+# Copy Project/Manifest and instantiate to install K8sDeputy and other dependencies...
+
+# Install K8sDeputy termination shim and dependencies (netcat and jq)
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y netcat-openbsd jq && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    julia --color=yes -e 'using K8sDeputy; K8sDeputy.install_supervise_shim("/usr/bin")'
```

Comment:
Since `nc` is installed even on minimal `alpine` images I think we can simply suggest this (assuming we bail on structured logging):
```suggestion
RUN curl -fsSLO https://raw.githubusercontent.com/beacon-biosignals/K8sDeputy.jl/refs/tags/v0.1.5/bin/supervise.sh && \
    install supervise.sh /usr/bin
```
We will need to mention `curl` will need to be installed

Reply:
it's not installed in the debian-slim images.

Reply:
Good point. I'm leaning towards just documenting what needs to be installed and only provide and example of using the `install_supervise_shim` function.